### PR TITLE
Hide context buttons during deployment

### PR DESCRIPTION
**Pull request by kartikgupta-db**
## Changes
* Hide buttons such as "open external url", "start cluster", "run resource" etc when deploying because these might be in inconsistent state. 
* Throw a better error from openExternal command when url is not found.

### CODE REVIEW

File: packages/databricks-vscode/package.json [hunk 1]
```diff
@@ -394,47 +394,47 @@
...
                 {
                     "command": "databricks.bundle.deployAndRun",
-                    "when": "view == dabsResourceExplorerView && viewItem =~ /^databricks.bundle.*.runnable.*$/",
+                    "when": "view == dabsResourceExplorerView && viewItem =~ /^databricks.bundle.*.runnable.*$/ && databricks.context.bundle.deploymentState == idle",
                     "group": "inline@0"
                 },
                 {
                     "command": "databricks.bundle.cancelRun",
-                    "when": "view == dabsResourceExplorerView && viewItem =~ /^databricks.bundle.*.cancellable.*$/",
+                    "when": "view == dabsResourceExplorerView && viewItem =~ /^databricks.bundle.*.cancellable.*$/ && databricks.context.bundle.deploymentState == running",
```
Comment (ilia-db) [inline packages/databricks-vscode/package.json:437]: `deploying`

 Maybe we can just have `bundle.deploying` flag? Or do we expect to have more than two deployment states?

Comment (kartikgupta-db) [reply packages/databricks-vscode/package.json:437]: Possibly error for enabling force?

**Review by fjakobs — APPROVED** [review]

File: packages/databricks-vscode/package.json (hunk 1 above)
Comment (ilia-db) [reply packages/databricks-vscode/package.json:433]: True. Also, the first part of my comment was about incorrect `running` state, is should be `deploying`, right?

Comment (kartikgupta-db) [reply packages/databricks-vscode/package.json:433]: Aah. Copilot 😅

Comment (kartikgupta-db) [reply packages/databricks-vscode/package.json:433]: https://github.com/databricks/databricks-vscode/pull/1064

**Pull request merged**